modules/top-level/output.nix

```diff
@@ -285,12 +285,21 @@ in
         '';
       };
 
-      extraConfigLuaPre = lib.mkIf config.wrapRc ''
-        -- Ignore the user lua configuration
-        vim.opt.runtimepath:remove(vim.fn.stdpath('config'))              -- ~/.config/nvim
-        vim.opt.runtimepath:remove(vim.fn.stdpath('config') .. "/after")  -- ~/.config/nvim/after
-        vim.opt.runtimepath:remove(vim.fn.stdpath('data') .. "/site")     -- ~/.local/share/nvim/site
-      '';
+      extraConfigLuaPre = lib.mkOrder 200 (
+        # Add a global table at start of init
+        ''
+          -- Nixvim's internal module table
+          -- Can be used to share code throughout init.lua
+          local _M = {}
+        ''
+        + lib.optionalString config.wrapRc ''
+
+          -- Ignore the user lua configuration
```

Comment:
Can you double check this does what you want? Nix's indented strings are weird and strip leading newlines, I can't recall if they only trim the _first_ `"\n"` or all of them...

Reply:
Sure, i'll double check real quick.

Reply:
![image](https://github.com/user-attachments/assets/b1c8d156-9244-4ee1-918e-20dc90317377)
LGTM

Reply:
I'll keep this in mind next time I want a leading newline. Thanks for checking!